### DISCUSSION TITLE
What happened?

### DISCUSSION THREAD

**Discussion by Smiles2go** [hape42/dailygammon#120] [General]
Not much going on here. Was collaboration a lost cause?

Suggestion to kill off iPhone variant was scary.

**Comment by hape42** [comment]
There don't seem to be many developers who have the time and desire to work on a "no profit project". So I am still the only developer. The chat is still the biggest problem. When the keyboard boots up, it is not very comfortable to write a message. Unfortunately, I made the decision years ago that everything should only work in landscape mode. Only with time did I realize that the portrait format also has many advantages. I've been working for some time on making the app usable in all directions. it's a lot of work and a lot of things have to be completely rethought. But once I've done that, a lot of changes will be much easier to implement. the chat in particular will be much easier to use.
